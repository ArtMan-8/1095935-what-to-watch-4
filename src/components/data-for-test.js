export const moviePoster = {
  title: `Seven Years in Tibet`,
  genre: `Drame`,
  year: `1997`,
};

export const films = [{
  id: `1184`,
  title: `We need to talk about Kevin`,
  poster: `img/we-need-to-talk-about-kevin.jpg`,
},
{
  id: `1134`,
  title: `What We Do in the Shadows`,
  poster: `img/what-we-do-in-the-shadows.jpg`,
},
{
  id: `1148`,
  title: `Revenant`,
  poster: `img/revenant.jpg`,
},
{
  id: `1194`,
  title: `Johnny English`,
  poster: `img/johnny-english.jpg`,
}];
