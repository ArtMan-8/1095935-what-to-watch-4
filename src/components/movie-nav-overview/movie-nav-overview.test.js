import React from 'react';
import renderer from 'react-test-renderer';
import MovieNavOverview from './movie-nav-overview.jsx';
import {films} from '../data-for-test.js';

const film = films[0];

describe(`MovieNavOverview`, () => {
  it(`Render MovieNavOverview`, () => {
    const tree = renderer.create(
        <MovieNavOverview
          rating={film.rating}
          votes={film.votes}
          description={film.description}
          director={film.director}
          starring={film.starring}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
