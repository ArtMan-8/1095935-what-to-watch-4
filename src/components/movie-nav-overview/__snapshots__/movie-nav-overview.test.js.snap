// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieNavOverview Render MovieNavOverview 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      10
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Awesome
      </span>
      <span
        className="movie-rating__count"
      >
        773
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Kevin's mother struggles to love her strange child, despite the increasingly dangerous things he says and does as he grows up. But Kevin is just getting started, and his final act will be beyond anything anyone imagined.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Lynne Ramsay
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Tilda Swinton, John C. Reilly, Ezra Miller
         and other
      </strong>
    </p>
  </div>,
]
`;
